RandomQuote/yashwanthvarma18/script.js

```diff
@@ -0,0 +1,20 @@
+const quoteText = document.getElementById('quoteText');
+const generateQuoteButton = document.getElementById('generateQuote');
+
+generateQuoteButton.addEventListener('click', getQuote);
+
+function getQuote() {
+  const settings = {
+    "async": true,
+    "crossDomain": true,
+    "url": "https://type.fit/api/quotes",
+    "method": "GET"
+  };
+
+  $.ajax(settings).done(function (response) {
```

Comment:
Good.

Reply:
Mostly I make some of them using Ai like Dale,Bing images ai , something i search for aesthetic 4k background landscape in Google.
@NitkarshChourasia